sycl/source/detail/buffer_impl.cpp

```diff
@@ -99,6 +100,34 @@ buffer_impl::getNativeVector(backend BackendName) const {
   addInteropObject(Handles);
   return Handles;
 }
+
+void buffer_impl::verifyProps(const property_list &Props) const {
+  auto CheckDataLessProperties = [](int PropertyKind) {
+#define __SYCL_DATA_LESS_PROP(NS_QUALIFIER, PROP_NAME, ENUM_VAL)               \
```

Comment:
nit: can we have a single lambda and pass it for to both parameter?

Reply:
how? I need to reconfigure __SYCL_DATA_LESS_PROP define.